docs/architecture/leo_storage.md

```diff
@@ -22,13 +22,17 @@ If its LeoStorage node finds inconsistency of an object, its node fixes the inco
 
 ## Data Structure
 
-LeoFS’ object consists of three layers which are **metadata**, **needle** and **object-container**.
+LeoFS' object consists of three layers which are as below:
 
-* **LeoObjectStorage** manages and stores both an object and metadata which stores as a needle.
-* LeoObjectStorage's metadata-storage handles and stores attributes of an object which includes filename, size, checksum, and others, and it depends on <a href="" target="_blank">Leveldb</a>.
-* LeoObjectStorage's object-container adopts a log structured file format, which is robust and high performance because an effect of the local file system is just a little part, and LeoStorage is necessary to remove unnecessary objects from the object containers, which is realized by the data compaction feature.
+* **Metadata** *(attributes of an object)* which is stored into **Metadata Store** and the header of a Needle
+* **Needle** which is a part of an AVS file
+* **AVS**[^1] which is an object-container of LeoStorage
 
-![](../assets/leofs-architecture.005.jpg)
+**LeoStorage** manages and stores both an object and metadata which are stored in AVS. Furthermore, metadata is redundantly stored in Metadata Store which utilizes Leveldb[^2] in order to be able to transform metadata to each other for durability.
```

Comment:
I think that main purpose utilizing Leveldb is to be able to faster look up by a `key` of an object and the reason why AVS stores not only objects but metadata(s) is for durability to enable LeoFS to recover Leveldb from AVS in case it's broken. so the current sentences look a little bit misreadable to me.

Reply:
@mocchira correct sentence is as below:

**LeoStorage** manages and stores both an object and metadata which are stored in AVS. Furthermore, metadata is redundantly stored in Metadata Store in order to be able to transform metadata to each other for durability.

Reply:
I'll fix it soon.

Reply:
LGTM.